src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
```

Comment:
**Issue**: Initializing absMax to 0 could lead to incorrect results if all numbers in the array are negative. For example, with the array [-1, -2, -3], the method would incorrectly return 0. <br> **Fix**: Initialize absMax with the first element of the array instead of 0. This ensures that the method correctly handles arrays with all negative numbers. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = numbers[0];
 ```